src/lib.rs

```diff
@@ -148,7 +148,7 @@
 //!     }
 //!
 //!     // Define additional `ErrorKind` variants. The syntax here is
-//!     // the same as `quick_error!`, but the `from()` and `cause()`
+//!     // the same as `impl_error_chain_kind!`, but the `from()` and `cause()`
```

Comment:
This doc appears on the generated struct, like for https://docs.rs/error-chain/0.10.0/error_chain/example_generated/struct.Error.html. I'm talking about being sure the users know what is and isn't allowed by the macro syntax.